test_files/vdlm2_3

```diff
@@ -8,7 +8,7 @@
 {"vdl2":{"app":{"name":"dumpvdl2","ver":"2.2.0"},"avlc":{"cmd":"Receive Ready","cr":"Response","dst":{"addr":"10422A","type":"Ground station"},"frame_type":"S","pf":false,"rseq":2,"src":{"addr":"A60852","status":"Airborne","type":"Aircraft"}},"burst_len_octets":13,"freq":136975000,"freq_skew":3.471563,"hdr_bits_fixed":0,"idx":0,"noise_level":-39.321873,"octets_corrected_by_fec":0,"sig_level":-6.352667,"station":"CS-KABQ-VDLM","t":{"sec":1659144838,"usec":285918}}}
 {"vdl2":{"app":{"name":"dumpvdl2","ver":"2.2.0"},"avlc":{"cmd":"Receive Ready","cr":"Response","dst":{"addr":"10422A","type":"Ground station"},"frame_type":"S","pf":false,"rseq":2,"src":{"addr":"A60852","status":"Airborne","type":"Aircraft"}},"burst_len_octets":13,"freq":136975000,"freq_skew":3.369126,"hdr_bits_fixed":0,"idx":0,"noise_level":-39.393845,"octets_corrected_by_fec":0,"sig_level":-6.422112,"station":"CS-KABQ-VDLM","t":{"sec":1659144844,"usec":378261}}}
-{"vdl2":{"app":{"name":"dumpvdl2","ver":"2.2.0"},"avlc":{"cmd":"XID","cr":"Command","dst":{"addr":"1012CA","type":"Ground station"},"frame_type":"U","pf":true,"src":{"addr":"A60852","status":"Airborne","type":"Aircraft"},"xid":{"err":false,"pub_params":[{"name":"param_set_id","value":"8885:1993"},{"name":"procedure_classes","value":[0,1]},{"name":"hdlc_options","value":[32,132,128]}],"type":"XID_CMD_HO","type_descr":"Handoff Initiation","vdl_params":[{"name":"param_set_id","value":"V"},{"name":"xid_sequencing","value":{"retry":0,"seq":0}},{"name":"dst_airport","value":"KLAS"},{"name":"ac_location","value":{"alt":37000,"loc":{"lat":35.0,"lon":-106.099998}}},{"name":"alternate_ground_stations","value":["10214A","10317A","10210A","104C2A","1020AA","1044AA","10355A","1015EA"]},{"name":"avlc_specific_options","value":[36]},{"name":"conn_mgmt","value":1}]}},"burst_len_octets":99,"freq":136975000,"freq_skew":3.483892,"hdr_bits_fixed":0,"idx":0,"noise_level":-39.235313,"octets_corrected_by_fec":0,"sig_level":-6.37737{"vdl2":{"app":{"name":"dumpvdl2","ver":"2.2.0"},"avlc":{"acars":{"ack":"!","blk_id":"8","crc_ok":true,"err":false,"flight":"WN1788","label":"Q0","mode":"2","more":false,"msg_num":"S20","msg_num_seq":"A","msg_text":"","reg":".N488WN"},"cr":"Command","dst":{"addr":"1012CA","type":"Ground station"},"frame_type":"I","poll":false,"rseq":0,"src":{"addr":"A60852","status":"Airborne","type":"Aircraft"},"sseq":0},"burst_len_octets":44,"freq":136975000,"freq_skew":3.724979,"hdr_bits_fixed":0,"idx":0,"noise_level":-39.161407,"octets_corrected_by_fec":0,"sig_level":-6.194024,"station":"CS-KABQ-VDLM","t":{"sec":1659144845,"usec":902123}}}
```

Comment:
Whilst I appreciate the clean up of the messages here, I left these obviously broken ones in here to ensure error handling was working

Reply:
Yeah, now that you mention it that makes sense. Reverted the commit.